_i18n/en/documentation/general/episode-deletion.md

```diff
@@ -0,0 +1,5 @@
+When you are done listening with an episode, in most cases you want to delete it from your phone, to free up space. There is different ways that you can do that.
+
+Firstly, you can of course manually delete episodes. When viewing the queue or a specific podcast, you can long-press an episode and tap `Delete`. When viewing the Completed tab in the Downloads screen you can also simply tap the bin icon. If you want, AntennaPod can automatically remove en episode from the queue if it is deleted. You can find this option under `Settings` » `Storage` »  `Delete removes from Queue`.
```

Comment:
```suggestion
Firstly, you can of course manually delete episodes. When viewing the queue or a specific podcast, you can long-press an episode and tap `Delete`. When viewing the `Completed` tab in the `Downloads` screen you can also simply tap the bin icon. If you want, AntennaPod can automatically remove en episode from the queue if it is deleted. You can find this option under `Settings` » `Storage` »  `Delete removes from Queue`.
```

Reply:
Hmm, not sure about this one. They're not buttons you tap to follow instructions, but rather describing a state. If we put backticks, shouldn't we also put them around 'Queue' and 'specific podcast'?

Reply:
You are right, let's keep it as it is